packages/node-client/lib/index.ts

```diff
@@ -879,21 +879,19 @@ export class Nango {
      * =======
      */
 
-    /**
-     * Triggers an action for a connection
-     * @param providerConfigKey - The key identifying the provider configuration on Nango
-     * @param connectionId - The ID of the connection for which the action should be triggered
-     * @param actionName - The name of the action to trigger
-     * @param input - An optional input data for the action
-     * @returns A promise that resolves with an object containing the response data from the triggered action
-     */
-
-    public async triggerAction<In = unknown, Out = object>(providerConfigKey: string, connectionId: string, actionName: string, input?: In): Promise<Out> {
+    private async _triggerAction<In = unknown, Out = object>(
```

Comment:
couldn't come with a better name for the private function. Suggestions are welcome.